src/ViewModels/InitializationViewModel.cs

```diff
@@ -13,9 +13,17 @@
 
 namespace DevHome.ViewModels;
 public class InitializationViewModel : ObservableObject
-{
+{
+#if CANARY_BUILD
```

Comment:
Not in this PR. Today we have a single `appsettings.json` we use for configuring Dev Home. We can also create `appsettings.Production.json`, `appsettings.Developement.json` and `appsettings.Canary.json` and store the corresponding PID and PFM in them. This will allow us to mock these values in unit test and decouple the compiler logic from the code. Resolving.